server/src/controllers/Products/ProductsIndexController.js

```diff
@@ -0,0 +1,34 @@
+const { products } = require('../../products.json');
+
+class IndexProductsController {
```

Comment:
ProductsIndexController

and the file should be named:
IndexController.js

Reply:
done